tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/modules/OSGiBundleListener.java

```diff
@@ -76,22 +76,21 @@ protected void handleTypedEvent( PropertyContentEvent event )
 
                 for( File file : files )
                 {
-                    try( JarFile jar = new JarFile( file ) ) {
-                        Enumeration<JarEntry> enu = jar.entries();
+                    JarFile jar = new JarFile( file );
```

Comment:
IF we don't use Try then we have to close teh JarFile.  This code never closes it (see the warning that Eclipse puts on that object).  Can you resend and send code that closes the JarFile?